dbos/_sys_db.py

```diff
@@ -1534,6 +1565,40 @@ def clear_queue_assignment(self, workflow_id: str) -> bool:
                     )
                 return True
 
+    T = TypeVar("T")
+
+    def call_function_as_step(self, fn: Callable[[], T], function_name: str) -> T:
```

Comment:
what for?

Reply:
This simplifies the code where we need to call system functions like getResult, list workflows, etc as steps in a workflow. Otherwise, we have to provide this wrapper everywhere.